### PR TITLE
Update zh-cn/web/html/element/u

### PR DESCRIPTION
**Pull request by AmeroHan**
### Description

- Update the content to fit the latest English version.
- Translate the text content in 2 examples’ HTML.
- Clarify the translated definition.

### CODE REVIEW

File: files/zh-cn/web/html/element/u/index.md [hunk 1]
```diff
@@ -161,10 +129,43 @@ cite {
...
-
 {{EmbedLiveSample("为 cite 元素添加样式", 650, 80)}}
 
+## 技术概要
+
+<table class="properties">
+ <tbody>
+  <tr>
+   <th scope="row"><a href="/zh-CN/docs/Web/HTML/Content_categories">内容类别</a></th>
+   <td><a href="/zh-CN/docs/Web/HTML/Content_categories#流式内容">流式内容</a>、<a href="/zh-CN/docs/Web/HTML/Content_categories#短语内容">短语内容</a>、palpable 内容。</td>
```
Comment (jasonren0403) [inline files/zh-cn/web/html/element/u/index.md:140]: https://developer.mozilla.org/zh-CN/docs/Web/HTML/Content_categories#%E5%8F%AF%E6%84%9F%E7%9F%A5%E5%86%85%E5%AE%B9

```suggestion
   <td><a href="/zh-CN/docs/Web/HTML/Content_categories#流式内容">流式内容</a>、<a href="/zh-CN/docs/Web/HTML/Content_categories#短语内容">短语内容</a>、可感知内容。</td>
```

File: files/zh-cn/web/html/element/u/index.md [hunk 2]
```diff
@@ -161,10 +129,43 @@ cite {
...
+    <th scope="row">隐式 ARIA 角色</th>
+    <td><a href="https://www.w3.org/TR/html-aria/#dfn-no-corresponding-role">无对应角色</a></td>
+  </tr>
+  <tr>
+   <th scope="row">允许的 ARIA 角色</th>
+   <td>任意</td>
+  </tr>
+  <tr>
+   <th scope="row">DOM 接口</th>
+   <td>{{domxref("HTMLElement")}} Gecko 1.9.2 (Firefox 4) 之前（包含），Firefox 为这个元素实现了 {{domxref("HTMLSpanElement")}} 接口。</td>
```
Comment (jasonren0403) [inline files/zh-cn/web/html/element/u/index.md:164]:
```suggestion
   <td>{{domxref("HTMLElement")}}</td>
```

File: files/zh-cn/web/html/element/u/index.md [hunk 3]
```diff
@@ -175,5 +176,5 @@ cite {
 
 ## 参见
 
-- {{ HTMLElement("span") }}，{{ HTMLElement("i") }}，{{ HTMLElement("em") }}，和 {{ HTMLElement("cite") }} 元素，取决于情况，可以替代使用。
-- CSS {{ cssxref("text-decoration") }} 属性可以用于完成 {{ HTMLElement("u") }} 元素的前面的视觉效果。
+- {{HTMLElement("span")}}、{{HTMLElement("i")}}、{{HTMLElement("em")}} 和 {{HTMLElement("cite")}} 元素，取决于情况，可以替代使用。
+- CSS {{cssxref("text-decoration")}} 属性可以用于完成 `<u>` 元素的上文提到的视觉效果。
```
Comment (jasonren0403) [inline files/zh-cn/web/html/element/u/index.md:180]: > The CSS {{cssxref("text-decoration")}} property should be used for non-semantic underlining.

```suggestion
- 应当使用 CSS {{cssxref("text-decoration")}} 属性添加非语义的下划线。
```

File: files/zh-cn/web/html/element/u/index.md [hunk 4]
```diff
@@ -175,5 +176,5 @@ cite {
 
 ## 参见
 
-- {{ HTMLElement("span") }}，{{ HTMLElement("i") }}，{{ HTMLElement("em") }}，和 {{ HTMLElement("cite") }} 元素，取决于情况，可以替代使用。
-- CSS {{ cssxref("text-decoration") }} 属性可以用于完成 {{ HTMLElement("u") }} 元素的前面的视觉效果。
+- {{HTMLElement("span")}}、{{HTMLElement("i")}}、{{HTMLElement("em")}} 和 {{HTMLElement("cite")}} 元素，取决于情况，可以替代使用。
```
Comment (jasonren0403) [inline files/zh-cn/web/html/element/u/index.md:179]: > The {{HTMLElement("span")}}, {{HTMLElement("i")}}, {{HTMLElement("em")}}, {{HTMLElement("b")}}, and {{HTMLElement("cite")}} elements should usually be used instead.

```suggestion
- 通常应使用 {{HTMLElement("span")}}、{{HTMLElement("i")}}、{{HTMLElement("em")}}、{{HTMLElement("b")}} 或 {{HTMLElement("cite")}} 元素代替。
```

**Review by jasonren0403 — CHANGES_REQUESTED** [review]
some suggestions

File: files/zh-cn/web/html/element/u/index.md [hunk 5]
```diff
@@ -5,88 +5,58 @@ slug: Web/HTML/Element/u
...
 要应用带下划线的外观而没有任何语义含义，请使用 {{cssxref("text-decoration")}} 属性的值 `underline`。
 
 ## 示例
 
 ### 表示拼写错误
 
-此示例使用`<u>`元素和一些 CSS 来显示包含拼写错误的段落，其中错误以红色波浪下划线样式表示，这种情况通常用于此目的。
+此示例使用 `<u>` 元素和一些 CSS 来显示包含拼写错误的段落，其中错误以通常用于此目的的红色波浪下划线样式表示。
 
 #### HTML
```
[the remark this answers is not in the document]
Comment (AmeroHan) [reply files/zh-cn/web/html/element/u/index.md:54]: 那我换回英文吧

**Review by jasonren0403 — APPROVED** [review]
👍 Thank you

**Pull request merged**